common/scala/src/main/scala/whisk/core/entity/Identity.scala

```diff
@@ -27,8 +27,15 @@ import whisk.core.database.MultipleReadersSingleWriterCache
 import whisk.core.database.NoDocumentException
 import whisk.core.entitlement.Privilege
 import whisk.core.entitlement.Privilege.Privilege
+import scala.util.Try
 
-protected[core] case class Identity(subject: Subject, namespace: EntityName, authkey: AuthKey, rights: Set[Privilege]) {
+case class UserLimits(invocationsPerMinute: Option[Int] = None, concurrentInvocations: Option[Int] = None, firesPerMinute: Option[Int] = None)
```

Comment:
I thought about this as well, but I figured this is more dynamic as in: If we add more namespace-definable limits in the future (and i expect us to) the schema is easily adapted to that vs. making the whole class `Option`al.